test/javascripts/controllers/preferences-account-test.js.es6

```diff
@@ -21,6 +22,10 @@ QUnit.skip("updating of associated accounts", function(assert) {
 
   assert.equal(controller.get("canUpdateAssociatedAccounts"), false);
 
+  controller.set("model.is_anonymous", false);
+
+  assert.equal(controller.get("canUpdateAssociatedAccounts"), false);
```

Comment:
If the mode is not anonymous, should we be able to update the associated accounts?

Reply:
Non-anonymous users are able to update associated accounts if other conditions are met https://github.com/discourse/discourse/pull/7311/files#diff-4aba2c210bc3e273a9f57c8348785fd4R109

Reply:
ah icic. There are other conditions 👍